app/views/catalog/_show_main_content.html.erb

```diff
@@ -19,14 +19,14 @@
   <script type="text/javascript">
     jQuery(document).ready(function(){
       var ctx = $("span.Z3988").attr("title");
-      var umlaut_base = "https://getit-dev.princeton.edu";
+      var umlaut_base = "https://getit.princeton.edu";
```

Comment:
Maybe in the [blacklight.yml](https://github.com/pulibrary/orangelight/blob/development/config/blacklight.yml)?